libs.bytelist/external/bytelist-1.0.2-license.txt

```diff
@@ -0,0 +1,95 @@
+Name: Bytelist
+Version: 1.0.2
+Description: The ByteList library from JRuby.
+License: EPL-v10
+Origin: https://github.com/jruby/bytelist
+Comment: This library was wrongly reported in Maven as released under the MIT license. It's licensed under the GPL 3, LGPL 3 or EPL 1.0 see: https://github.com/jruby/bytelist/commit/9a0d9de4d21f6fc4342f946166d43cf3bf8b798a for the upgrade.
```

Comment:
This assessment is not correct. You are are referring to git master and that is indeed triple licensed as described. That is not true for version 1.0.2:

https://github.com/jruby/bytelist/blob/4e41c255c4d58328e3a422fc79651a0464b55600/src/org/jruby/util/ByteList.java

This is CPL 1.0 + GPL 2.0 + LGPL 2.1

I would do this:
- move to 1.0.15
- document, that the POM info is wrong (https://github.com/jruby/bytelist/commit/9a0d9de4d21f6fc4342f946166d43cf3bf8b798a#diff-600376dffeb79835ede4a0b285078036)
- add info, that the sources file next to the binary file clearly states the correct EPL/GPL/LGPL triple license
- bump specification version (0.30.0) as API got bigger

I ran the two jars through japi-compliance-checker and the result was really positive. The attached zip contains the HTML report and bytelist gained new methods, but kept the old ones.

[compat_report.html.zip](https://github.com/apache/incubator-netbeans/files/1549356/compat_report.html.zip)